### PR TITLE
Fixing company metrics api service

### PR DESCRIPTION
**Pull request by mnycz-ch**

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/pscstatementdataapi/api/CompanyMetricsApiService.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import uk.gov.companieshouse.api.model.ApiResponse;
+import uk.gov.companieshouse.logging.Logger;
+
+import java.util.Optional;
+
+
+@Service
+public class CompanyMetricsApiService {
+
+    private static final String GET_COMPANY_METRICS_ENDPOINT = "/company/%s/metrics";
```
Comment (dgurjar-ch) [inline src/main/java/uk/gov/companieshouse/pscstatementdataapi/api/CompanyMetricsApiService.java:21]: please move end point from properties or yml file

**Review by jaddison-ch — APPROVED** [review]

File: src/main/java/uk/gov/companieshouse/pscstatementdataapi/api/CompanyMetricsApiService.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        ));
+        final InternalApiClient internalApiClient = this.apiClientService.getInternalApiClient();
+        PrivateCompanyMetricsGet companyMetrics =
+                internalApiClient.privateCompanyMetricsResourceHandler()
+                        .getCompanyMetrics(
+                                String.format(GET_COMPANY_METRICS_ENDPOINT, companyNumber));
+        try {
+            ApiResponse<MetricsApi> execute = companyMetrics.execute();
+            return Optional.ofNullable(execute.getData());
+        } catch (URIValidationException exp) {
```
Comment (dgurjar-ch) [inline src/main/java/uk/gov/companieshouse/pscstatementdataapi/api/CompanyMetricsApiService.java:55]: Please print exception in log for better trace

**Review by akunati-ch — APPROVED** [review]

File: src/main/java/uk/gov/companieshouse/pscstatementdataapi/api/CompanyMetricsApiService.java (hunk 1 above)
Comment (mnycz-ch) [reply src/main/java/uk/gov/companieshouse/pscstatementdataapi/api/CompanyMetricsApiService.java]: done

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by dgurjar-ch — APPROVED** [review]

File: src/main/java/uk/gov/companieshouse/pscstatementdataapi/api/CompanyMetricsApiService.java (hunk 2 above)
Comment (mnycz-ch) [reply src/main/java/uk/gov/companieshouse/pscstatementdataapi/api/CompanyMetricsApiService.java:55]: done

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**